### ISSUE TITLE
Model not converging on 224x224

### ISSUE THREAD

**Issue by devmaha** [brain-research/acai#13]
Dear code author, it's really great to see that you have answered all queries till now. I was wondering if you could kindly help me a bit too.
I am trying to train ACAI on 224x224 size images with following args but after 50-70 epochs the loss becomes constant, and even after 1000+ epochs the images created are hazy. I have played with various learning rates, latent size etc. but nothing helped. Could you please guide how we can train larger size (and possibly colored) images with this model.

args = {
    'epochs': 10000,
    'width': 224,
    'latent_width': 10,
    'depth': 16,
    'advdepth': 16,
    'advweight': 0.5,
    'reg': 0.2,
    'latent': 2,
    'colors': 1,
    'lr': 0.01,
    'batch_size': 64,
    'device': 'cuda'
}

Appreciate you giving your time to read this.

**Comment by craffel** [comment]
Sorry, we have not experimented with larger image sizes. It's not obvious what hyperparameter changes are necessary for your case. Good luck!
